maintenance/recipes/turn_on.rb

```diff
@@ -0,0 +1,11 @@
+execute "update maintenance estimated time" do
```

Comment:
Can we replace it with `template`?
https://github.com/sdtechdev/frt-opsworks-cookbooks/pull/38#discussion_r555793425

Reply:
Updated!